pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2736,6 +2736,18 @@ self: super: {
       sha256 = "sha256-ZdCXMhni+RGisRODiElObW5c4hKy2giWQmWnatqeRJo=";
     }))
   ];
+
+  # jailbreak to allow rerefined >=0.7.0
+  binrep = doJailbreak super.binrep;
+
+  # jailbreak to allow deepseq >= 1.5
+  blucontrol = doJailbreak super.blucontrol;
+
+    # jailbreak to allow base >= 4.19
+  taskwarrior = doJailbreak super.taskwarrior;
+
+  # jailbreak to allow network >= 3.2
+  zre = doJailbreak super.zre;
```

Comment:
I've just uploaded a fix for this. I didn't notice it in the parent PR (sorry and thanks!). It should get fixed tomorrow so no need for jailbreak (but I don't mind reverting it myself later).

Reply:
thanks 👍 I'll drop the commit from this PR then